CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Fixed
+- Fixes coarse field sampling in `PolySlab` VJP causing inaccurate gradients for polygons with long edges in some cases.
```

Comment:
maybe just "Finer field sampling along `PolySlab` edges and slab axis for more accurate gradients."?